okhttp/src/main/java/okhttp3/internal/http2/Hpack.java

```diff
@@ -370,6 +370,7 @@ ByteString readByteString() throws IOException {
     private static final int SETTINGS_HEADER_TABLE_SIZE_LIMIT = 16384;
 
     private final Buffer out;
+    private final boolean useCompression;
```

Comment:
Turning on Huffman encoding by default broke a number of tests because most of the string literals can be encoded, so I pulled it out into a boolean. It wasn't clear to me if there was any value in updating all those tests to now use the Huffman encoding, rather than the raw literals.